website/content/docs/secrets/rabbitmq.mdx

```diff
@@ -68,6 +68,25 @@ management tool.
     more about [RabbitMQ management tags][rmq-perms]
     and [RabbitMQ topic authorization][rmq-topics].
 
+Similarly, if a static username is required, one can configure a static role
+instead:
+
+1.  Configure a role that maps a name in Vault to virtual host permissions:
+
+    ```text
+    $ vault write rabbitmq/static-roles/my-static-role \
+        username="foobar" \
+        rotation_period=300 \
+        vhosts='{"/":{"write": ".*", "read": ".*"}}'
+    Success! Data written to: rabbitmq/static-roles/my-static-role
+    ```
+
+    By writing to the `static-roles/my-static-role` path we are defining the
+    `my-static-role` static role. The way this role is created is similar to
+    the role in the example above. However, it will always keep `foobar` as a
+    username, and automatically rotate the password every 300 seconds. Hence
+    it will not create a new password every time credentials are requested.
```

Comment:
```suggestion
Writing to the `static-roles/my-static-role` defines a static role called
`my-static-role`. The creation process is similar to the previous example,
but the role keeps `vhost` as a username and automatically rotate the
password every 300 seconds rather than creating a new password every time
Vault receives a credential request.
```

style correction: active voice